### ISSUE TITLE
V fmt breaks custom module imports

### ISSUE THREAD

**Issue by Cons-Cat** [vlang/v#11058]
**V version:** V 0.2.2 aca66d5.f9c279d
**OS:** Void Linux

**What did you do?**
Wrote a module, then imported it in a V file one directory up, such as:
```v
import mymodule
```

**What did you expect to see?**
This causes no issues, as the code itself compiles and runs correctly.
 
**What did you see instead?**
V fmt turns that code into
```v
import  as mymodule
```

**Comment by serkonda7** [comment]
Can you give more details please. I cannot reproduce this on Manjaro.

**Comment by Cons-Cat** [comment]
> Can you give more details please, like the exact command used to run fmt. I cannot reproduce this on Manjaro.

This has happened on every "real" V project I've worked on, so I lazily didn't bother investigating in detail. Upon trying to trivially reproduce the issue, I see it isn't quite as simple as I thought, so that's totally my fault. So far, I can only reproduce it by trying to import a cloned project.

Here are steps:
```sh
mkdir example/
cd example/
git clone https://github.com/cons-cat/kitty_v.git kitty/
echo 'module main 
import kitty
fn main() {
    println('hi')
}' >> main.v
v fmt main.v
```
The output is:
```v
module main

import  as kitty

fn main() {
	println(hi)
}
```
This issue is not unique to this specific repository. The same problem occurs in, for example, ``https://github.com/cons-cat/v-xml.git``, if more examples help. Note that in my xml repo, the module is not at the top of the directory, though.

**Comment by JalonSolov** [comment]
Try again with one modification to your example:  between the `echo` and the `v fmt` command, add `touch v.mod`.

When you do that, it makes it work.

It also works if you either clone `kitty` into the `.vmodules` folder, or if you creates a link - instead of `touch v.mod` do `ln -s ${PWD}/kitty ~/.vmodules`

In other words, the bug is that `vfmt` can't find the module, and instead of complaining that it can't find it, it is doing the weird transformation on the `import` statement.

**Comment by serkonda7** [comment]
Fix is in #11192.
I will leave this issue open until I've added a proper test.
